include/drivers/msgbox/sunxi-msgbox.h

```diff
@@ -7,7 +7,8 @@
 #define DRIVERS_MSGBOX_SUNXI_MSGBOX_H
 
 #include <clock.h>
-#include <msgbox.h>
```

Comment:
All of the other driver-specific headers include their generic class header, so I'd prefer this to be left as-is.

Reply:
OK, I wasn't sure about this, and it wasn't actually used, but I understand the rationale.